bin/spiced/src/lib.rs

```diff
@@ -110,7 +111,13 @@ pub async fn run(args: Args) -> Result<()> {
 
     let rt: Runtime = Runtime::builder()
         .with_app_opt(app)
+        // User configured extensions
         .with_extensions(extension_factories)
+        // Extensions that will be auto-loaded if not explicitly loaded and requested by a component
+        .with_autoload_extensions(HashMap::from([(
+            "spice_cloud".to_string(),
```

Comment:
that should be under the feature flag `"spice-cloud"`

Reply:
Removed